docs/HOWTO-create-backup-for-new-database-creation.md

```diff
@@ -6,9 +6,43 @@ When standing up a new database instance, the project team must submit that data
 
 These are the current requirements for submitting a database backup to the Hack Oregon DevOps squad that can be successfully used to perform the initial load of a new database instance.
 
-* Coordinate with the Hack Oregon DevOps squad to finalize the name of the database instance and acquire the username that will own that database instance
-* Generate a `pgdump` backup using one of two methods:
-* * the procedure that results in `.backup` file in native pgdump format
-* * the procedure that generates "raw SQL" data into a `.sql` file
+1. Coordinate with the Hack Oregon DevOps squad to finalize the name of the database instance and acquire the username that will own that database instance (e.g. a project named mars-space-flight might have two databases, mars-space-flight-surface-lander and mars-space-flight-liftoff).
 
+
+2. The current required format for database backups is compressed plain-text, and the current supported version of PostgreSQL at this time is 9.6.6 (latest supported on Amazon Linux 2). Other formats are sometime incompatible with PostgreSQL servers at the same release or newer. Note that database backups are not designed to be restored to older versions of PostgreSQL - e.g. don't take a backup from 10.3 and expect it to restore to 9.6.
+
+    A plain-text backup is SQL code with some `psql` commands mixed in, You can read it with a text editor. A compressed plain text backup can be read if you uncompress it first.
+3. A database on the source server should have the same owner name as it will have on the destination server. All the objects in the database should have that owner name too. This means the user / role must exist on the source when the backup is created and on the destination before it is restored.
+
+    You can change owners of databases and the objects they contain with SQL script like the following:
+
+    ```
+    ALTER DATABASE odot_crash_data OWNER TO "transportation-systems";
+    REASSIGN OWNED BY postgres TO "transportation-systems";
```

Comment:
Change "postgres" to "znmeb" - it turns out you can't reassign an object owned by "postgres"